remappings.txt

```diff
@@ -1,10 +1,9 @@
-@chainlink=lib/chainlink/
+@chainlink=node_modules/@chainlink/
 @juicebox=contracts/
-@openzeppelin/contracts/=lib/openzeppelin-contracts/contracts/
-@paulrberg/contracts/math/=lib/prb-math/contracts/
-chainlink/=lib/chainlink/
+@openzeppelin/contracts/=node_modules/@openzeppelin/contracts/
```

Comment:
I think we can even remove the `node_modules/` and just add it in the toml (`libs=['lib', 'node_modules']`)

Reply:
@xBA5ED wdyt?

Reply:
Good call, quite a lot cleaner now